components/treeview/index.css

```diff
@@ -11,16 +11,19 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-TreeView {
-  --spectrum-treeview-line-height: 1.2;
+  --spectrum-treeview-line-height: 1.214285;
```

Comment:
Ooh. This looks like a magic number ✨ Is it worth a comment about how we got to it?

Reply:
I've added a comment about this (it's 16

Reply:
I've added a comment about this. It's to get the expected 17px line-height; 17 (line-height) divided by 14 (font size).